index.html

```diff
@@ -2250,6 +2329,36 @@ <h1>Core Representations</h1>
 for more information.
     </p>
 
+    <section>
+      <h2>Creating Representations</h2>
+      <p>
+The <a href="#data-model">data model</a> provided in this specification
+supports being serialized into a variety of existing representations. Some
+applications might require the creation of a new representation. All
+representations require the following:
+      </p>
+      <ol>
+        <li>
+A representation MUST define an unambiguous encoding and decoding for all
+property names and their associated values as defined in this specification.
```

Comment:
This should be defined in terms of the types associated with the property values and not the values themselves. It's a subtle difference, but it allows us to continue to talk about things in terms of property definitions.

Reply:
Agree. I just moved the text in this PR and expect that we'll make the change you suggest in a subsequent PR, when the "Representation Requirements" section is updated with rewording / new rules that we might have missed (like how you register a representation in the DID Spec registries).